<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureDialog</class>
 <widget class="QDialog" name="ConfigureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>301</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Asynchronous One Buffered AI with Trigger Delay to Stop - Configuration</string>
  </property>
  <widget class="QComboBox" name="cmbDevice">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>260</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDevice">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>91</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string>Device:</string>
   </property>
   <property name="buddy">
    <cstring>cmbDevice</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>260</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="grpAISetting">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>271</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Buffered AI settings</string>
   </property>
   <widget class="QComboBox" name="cmbChannelCount">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="lblChannelCount">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>60</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Channel count:</string>
    </property>
    <property name="buddy">
     <cstring>cmbChannelCount</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="lblSectionLength">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>120</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Section Length:</string>
    </property>
    <property name="buddy">
     <cstring>cmbValueRange</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="lblRateUnit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Hz</string>
    </property>
    <property name="buddy">
     <cstring>cmbValueRange</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbValueRange">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbChannelStart">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtClockRatePerChan">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblValueRange">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>90</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Value range:</string>
    </property>
    <property name="buddy">
     <cstring>cmbValueRange</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="lblClockRate">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>150</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Clock rate:</string>
    </property>
    <property name="buddy">
     <cstring>cmbValueRange</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtSectionLength">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1024</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblChannelStart">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Channel start:</string>
    </property>
    <property name="buddy">
     <cstring>cmbChannelStart</cstring>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpTriggerSetting">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>281</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Trigger settings</string>
   </property>
   <widget class="QComboBox" name="cmbTriggerEdge">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="lblTriggerEdge">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Edge:</string>
    </property>
    <property name="buddy">
     <cstring>cmbChannelCount</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="lblTriggerDelayCount">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Delay count:</string>
    </property>
    <property name="buddy">
     <cstring>cmbValueRange</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="lblTriggerLevelUnit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>21</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>21</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>V</string>
    </property>
    <property name="buddy">
     <cstring>cmbValueRange</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbTriggerSource">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTriggerLevel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblTriggerLevel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Trigger level:</string>
    </property>
    <property name="buddy">
     <cstring>cmbValueRange</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDelayCount">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>500</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblTriggerSource">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Source:</string>
    </property>
    <property name="buddy">
     <cstring>cmbChannelStart</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="lblTriggerIndex">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Trigger:</string>
    </property>
    <property name="buddy">
     <cstring>cmbChannelStart</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbTriggerTriggerIndex">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblProfilePath">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>91</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string>Profile:</string>
   </property>
   <property name="buddy">
    <cstring>cmbDevice</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtProfilePath">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBrowse">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>cmbDevice</tabstop>
  <tabstop>cmbChannelStart</tabstop>
  <tabstop>cmbChannelCount</tabstop>
  <tabstop>cmbValueRange</tabstop>
  <tabstop>edtSectionLength</tabstop>
  <tabstop>edtClockRatePerChan</tabstop>
  <tabstop>btnOK</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
